docker/bin/zookeeperStart.sh

```diff
@@ -20,6 +20,10 @@ MYID_FILE=$DATA_DIR/myid
 LOG4J_CONF=/conf/log4j-quiet.properties
 DYNCONFIG=$DATA_DIR/zoo.cfg.dynamic
 
+# Copy over the zoo.cfg file in the configmap to make it writable
+ZOOCFGDIR=$DATA_DIR
+cp /conf/zoo.cfg /data/zoo.cfg
```

Comment:
I don't have any major concern with this change, but for organization purposes, we may want to have the configuration files in a path that ends with `conf/`. For example, we could have:

1-  a common root `zookeeper`
2- the data dir being `zookeeper/data`
3- the conf dir being `zookeeper/conf`

Reply:
Sounds reasonable. I put the config files into the '/conf' in the '/data' for now. Since `/data` is the mount point of a pvc, changing the name will require a new release for the operator. Right now the PR only requires the release for zookeeper image.

Reply:
If you agree that this is a change we want to do in the future, then do you want to create an issue to track it?

Reply:
I don't think moving configuration to /data directory is a good solve to this problem. At some point if we need to purge data folder , this will create issues. It could be a reasonable temporary workaround  though. Configurations should be maintained under a separate 'conf' folder, which could be "/conf" or "/zookeeper/conf"
Please see my other updates to Issue #66.

Reply:
Got it. Changed to the corresponding structure. Please review it.